api/src/main/resources/messages_fr.properties

```diff
@@ -0,0 +1 @@
+tag.title=
```

Comment:
yes. taken care of this and deleted all the classes, and gsp files that weren't needed